pkgs/development/python-modules/hypothesis.nix

```diff
@@ -21,7 +21,7 @@ buildPythonPackage rec {
     sha256 = "1s911pd3y9hvk0hq2fr6i68dqv1ciagryhgp13wgyfqh8hz8j6zv";
   };
 
-  checkInputs = stdenv.lib.optionals doCheck [ pytest flake8 flaky ];
+  checkInputs = stdenv.lib.optionals doCheck [ pytest pytest_xdist flake8 flaky ];
```

Comment:
If I'm correct that was only with pytest 3.1.0. I reverted to pytest 3.0.7 because it broke a lot of packages.